src/config.js

```diff
@@ -241,6 +241,71 @@ const filecoin = {
   addedBy: 'Mostafa Balata',
 };
 
+const viberate = {
+  tokenContract: '0x2c974b2d0ba1716e644c1fc59982a89ddd2ff724',
+  information: {
+    aliasName: 'Viberate',
+    website: 'https://www.viberate.com/',
+    logo: 'https://viberateassets.azureedge.net/favicon.ico',
+  },
+  events: {
+    Mint :{
+      args: {
+        tokens: '_value',
+        sender: '_to',
+      },
+      firstTransactionBlockNumber: 4240691,
+      lastTransactionBlockNumber:  4348935,
```

Comment:
This is still invalid block number. Look at the results! must be block before developers claim their tokens. You may use your new tool to find this block ;>